requirements/extras/confluentkafka.txt

```diff
@@ -1 +1 @@
-confluent-kafka>=2.2.0
+confluent-kafka>=2.2.0 ; python_version < "3.13"
```

Comment:
```suggestion
confluent-kafka>=2.2.0
```

Reply:
Let’s see if we can remove it already

Reply:
Didn’t work:
```console
Failed to build confluent-kafka
```
https://github.com/celery/kombu/actions/runs/10981646035/job/30488936527